trace/trace.proto

```diff
@@ -105,38 +105,31 @@ message Span {
       Attributes attributes = 2;
     }
 
-    // An event describing an RPC message sent or received on the network.
-    message NetworkEvent {
-      // For sent messages, this is the time at which the first bit was sent.
-      // For received messages, this is the time at which the last bit was
-      // received.
-      google.protobuf.Timestamp time = 1;
-
-      // Indicates whether the network message was sent or received.
+    // An event describing a message sent/received between Spans.
+    message MessageEvent {
+      // Indicates whether the message was sent or received.
       enum Type {
         // Unknown event type.
         TYPE_UNSPECIFIED = 0;
-        // Indicates a sent RPC message.
+        // Indicates a sent message.
         SENT = 1;
-        // Indicates a received RPC message.
-        RECV = 2;
+        // Indicates a received message.
+        RECEIVED = 2;
       }
 
-      // Type of NetworkEvent. Indicates whether the RPC message was sent or
+      // Type of MessageEvent. Indicates whether the message was sent or
       // received.
-      Type type = 2;
+      Type type = 1;
 
-      // An identifier for the message, which must be unique in this span and
-      // the same between RECV and SENT (this allows to identify a message
-      // between the sender and receiver). For example, this could be a
-      // sequence id.
-      uint64 message_id = 3;
+      // An identifier for the message, which must be unique in this span.
+      uint64 id = 2;
```

Comment:
Can there be more than one MessageEvent per message, within one span?

Reply:
Yes. Think about a stream request/response we record one entry for each part of the stream.

Reply:
I think it would help to explain that in the comment, like the comment on master.  The comment in the PR could be misread as saying that the MessageEvent's id must be unique, especially since MessageEvent is also a Protocol Buffer message.

Reply:
Can you be more explicit how do you want me to rephrase. Happy to change.

Reply:
Good suggestion. Thanks.